app/build.gradle

```diff
@@ -11,12 +12,45 @@ final ERROR_MESSAGE = "You have not defined an app key for the Onfido api."
 
 def apiKey = getValueFromFileOrEnv(APP_TOKEN_ENV_VAR_NAME, APP_TOKEN_FILE, APP_TOKEN_KEY, ERROR_MESSAGE)
 
-def sdkVersion = '3.0.0'
-def versionCodeNumber = 11
+def sdkVersion = '4.0.0'
+def versionCodeNumber = 400
+
+buildscript {
+    repositories {
+        jcenter()
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'com.getkeepsafe.dexcount:dexcount-gradle-plugin:0.6.4'
+    }
+}
+
+dexcount {
+    format = "list"
+    includeClasses = false
+    includeFieldCount = true
+    includeTotalMethodCount = true
+    orderByMethodCount = false
+    verbose = false
+    maxTreeDepth = Integer.MAX_VALUE
+    teamCityIntegration = false
+    enableForInstantRun = false
+    teamCitySlug = null
+    runOnEachAssemble = true
+    maxMethodCount = 120000
+}
 
 android {
+    signingConfigs {
+        release {
+            keyAlias 'release'
+            keyPassword 'release'
+            storeFile file('/Users/sergiomoura/development/release.jks')
+            storePassword 'release'
+        }
+    }
     compileSdkVersion 27
-
+    buildToolsVersion '27.0.3'
```

Comment:
I think it's not required anymore, is it?

Reply:
you're right, removed